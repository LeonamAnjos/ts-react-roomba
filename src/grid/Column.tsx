import React, { ReactNode } from "react";
import "../styles/Grid.css";

type ColumnProps = {
  children?: ReactNode | undefined;
};

const Column = ({ children }: ColumnProps) => {
  return <div className="Column">{children}</div>;
};

export default Column;
