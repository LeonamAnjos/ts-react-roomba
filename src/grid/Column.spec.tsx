import { render } from "@testing-library/react";
import Column from "./Column";

describe("<Column />", () => {
  it("SHOULD render", () => {
    const { container } = render(<Column>{"Content Test"}</Column>);

    expect(container).toMatchSnapshot();
  });
});
