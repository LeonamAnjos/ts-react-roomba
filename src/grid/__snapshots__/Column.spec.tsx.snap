// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> SHOULD render 1`] = `
<div>
  <div
    class="Column"
  >
    Content Test
  </div>
</div>
`;
